com.unity.render-pipelines.high-definition-config/Runtime/ShaderConfig.cs

```diff
@@ -4,40 +4,67 @@
 
 namespace UnityEngine.Rendering.HighDefinition
 {
+    /// <summary>
+    /// Probe Volume Evaluation Mode.
+    /// </summary>
     [GenerateHLSL(PackingRules.Exact)]
     public enum ProbeVolumesEvaluationModes
     {
+        /// <summary>Probe volumes are disabled.</summary>
         Disabled = 0,
+        /// <summary>Probe volumes are evaluated in the light loop.</summary>
         LightLoop = 1,
+        /// <summary>Probe volumes are evaluated in the material pass.</summary>
         MaterialPass = 2,
     }
 
+    /// <summary>
+    /// Encoding of Probe Volumes.
+    /// </summary>
     [GenerateHLSL(PackingRules.Exact)]
     public enum ProbeVolumesEncodingModes
     {
+        /// <summary>Encoded using L0 spherical harmonics.</summary>
         SphericalHarmonicsL0 = 0,
+        /// <summary>Encoded using L1 spherical harmonics.</summary>
         SphericalHarmonicsL1 = 1,
+        /// <summary>Encoded using L2 spherical harmonics.</summary>
         SphericalHarmonicsL2 = 2
     }
 
+    /// <summary>
+    /// Probe Volume bilateral filtering mode.
```

Comment:
Options for the mode HDRP uses for probe volume bilateral filtering.

Reply:
Originally thought:
Options for the mode HDRP uses to bilaterally filter probe volumes.
but I'm not sure that conveys the same meaning. If you think it does, use this instead,